packages/ui/src/council/types/PastCouncil.ts

```diff
@@ -37,21 +39,28 @@ export const getTotalSpent = (spendingEvents: CouncilSpendingEventFieldsFragment
 
 export const getSpentOnProposals = (fundingRequests: FundingRequestApprovedFragment[]) => {
   return fundingRequests.reduce((sum, fundingRequest) => {
-    const details = asProposalDetails(fundingRequest.proposal.details as DetailsFragment) as FundingRequestDetails
+    const details = asProposalDetails(fundingRequest.details as DetailsFragment) as FundingRequestDetails
     const amount = details.destinations?.reduce((a, b) => a.add(b.amount), BN_ZERO) || BN_ZERO
 
     return sum.add(amount)
   }, BN_ZERO)
 }
 
 export const asPastCouncilWithDetails = (
+  workingGroupRewardPaidEvents: PastCouncilRewardPaidEventFieldsFragment[],
+  workingGroupBudgets: PastCouncilBudgetUpdatedEventFieldsFragment[],
   councilFields: PastCouncilDetailedFieldsFragment,
   spendingEvents: CouncilSpendingEventFieldsFragment[],
   fundingRequestsApproved: FundingRequestApprovedFragment[]
 ): PastCouncilWithDetails => {
   return {
     ...asPastCouncil(councilFields),
-    totalSpent: getTotalSpent(spendingEvents),
+    // totalSpent: getTotalSpent(spendingEvents),
+    totalSpent: councilFields.councilMembers
+      .reduce((a, b) => a.add(new BN(b.accumulatedReward)), BN_ZERO)
+      .add(workingGroupBudgets.reduce((a, b) => a.add(new BN(b.budgetChangeAmount)), BN_ZERO))
+      .add(getSpentOnProposals(fundingRequestsApproved))
+      .add(workingGroupRewardPaidEvents.reduce((a, b) => a.add(new BN(b.amount)), BN_ZERO)),
```

Comment:
Because this code will probably have to be updated in the furture I think it's worth makin it a bit clearer so please remove the `export` from `getTotalSpent` and update it's definition to something like:
```js
const getTotalSpent = (
  councilMembers: ...,
  workingGroupBudgets: ...,
  workingGroupRewardPaidEvents: ...,
  fundingRequestsApproved: ...,
) => {
  const totalAccumulatedReward = councilMembers
    .reduce((a, b) => a.add(new BN(b.accumulatedReward)), BN_ZERO)
  const totalBudgetChange = workingGroupBudgets
    .reduce((a, b) => a.add(new BN(b.budgetChangeAmount)), BN_ZERO)
  const totalRewardPaid = workingGroupRewardPaidEvents
    .reduce((a, b) => a.add(new BN(b.amount)), BN_ZERO)
  const spentOnProposal = getSpentOnProposals(fundingRequestsApproved)

  return totalAccumulatedReward
    .add(totalBudgetChange)
    .add(totalRewardPaid)
    .add(spentOnProposal)
}
```
Then:
```suggestion
    totalSpent: getTotalSpent(councilFields.councilMembers, workingGroupBudgets, workingGroupRewardPaidEvents, spentOnProposal),
```

Also @chrlschwb should channel payout be included here ?

Reply:
Yes, I believe channel payout should be included